app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
+    <div class="inset">
+      <p>Becoming a secondary teacher is an excellent career choice for people who want to make an impact. Your expertise and love for your subject could spark the interests of your pupils to follow in your footsteps. Beyond the classroom, you'll also play a crucial role in building pupils' confidence and supporting their personal growth.</p>
```

Comment:
```suggestion
      <p>Becoming a secondary school teacher is an excellent career choice if you want to have a positive impact on young people's lives. Your expertise and love for your subject could spark the interests of your pupils to follow in your footsteps. Beyond the classroom, you'll also play a crucial role in building pupils' confidence and supporting their personal growth.</p>
```

Reply:
Not sure about the 'follow in your footsteps bit' - is this for the pupils to teach the subject or for them to explore the subject more? 

Wonder if it would be good to add a bit here about making learning engaging for pupils, then it doesn't have to be subject specific. 

So something like 'You can engage and motivate pupils to develop a passion for learning."